css/app.css

```diff
@@ -417,10 +417,10 @@
 }
 
 .CodeMirror-hint-information .infoType {
-  color: #30a;
+  color: #CA9800;
```

Comment:
Most of the rest of the file is uppercase hex colors, so I'll leave this for now - but we should definitely lint and unify around some standard